config/initializers/bugsnag.rb

```diff
@@ -0,0 +1,3 @@
+Bugsnag.configure do |config|
+  config.api_key = ENV["BUGSNAG_API_KEY"]
```

Comment:
This does nothing in development and test, but configures error reporting in production where the env is set.